homeassistant/components/unifiled/light.py

```diff
@@ -57,6 +57,7 @@ def __init__(self, light, api):
         self._name = light["name"]
         self._unique_id = light["id"]
         self._state = light["status"]["output"]
+        self._available = self._api.getlightavailable(self._unique_id)
```

Comment:
Is it a cached value that's returned?

Reply:
It is not a cached value, Replaced it with a way of getting a cached one.